src/core/Address.ts

```diff
@@ -28,15 +28,13 @@ export abstract class Address {
 
     /**
      * Ge address bytes
-     *
-     * @returns {Uint8Array}
      */
     public abstract getAddressBytes(): Uint8Array;
 
     /**
      * Get address in the encoded format ex: NAR3W7B4BCOZSZMFIZRYB3N5YGOUSWIYJCJ6HDFH.
      *
-     * @returns {string} Encoded addres string
+     * @returns Encoded addres string
```

Comment:
is there a way to add spell checker :D 
and enforce periods (or not)?

Reply:
😄 , will have a look

Reply:
the spell checker plugin we had before was not smart enough, it flags lots reserved words

Reply:
I use CSpell and Language Tool with VSCode, and add keywords to my personal dictionary. After a while they know all of them :)

Reply:
I agree with @Jaguar0625 

I think like the style checker and style formatter, having a ready-to-run cli spellcheck is super nicer. If not everybody would have their own dictionary and tools that may not be in sync (different IDEs, different reserved words) and cannot be validated in PRs.

We should somehow have a shared dictionary and add the spellchecker to each build tool. 
One example https://www.npmjs.com/package/eslint-plugin-spellcheck.

There are going to be a lot of typos, but well, need to fix them

Reply:
@segfaultxavi can you share your dictionary? So we all use the same words. We probably need to push that somewhere in github and each language should use the right tool for the spellchecking

Reply:
Sure... https://hackmd.io/lrT8XcxyRcapPCb7z_idEQ?view

Reply:
@Jaguar0625 I have craeted https://github.com/symbol/sdk-typescript/issues/37. Does it make sense to fix this in another PR?